public/static/docs/command-reference/diff.md

```diff
@@ -1,41 +1,53 @@
 # diff
 
-Show changes between commits in the <abbr>DVC repository</abbr>, or between a
-commit and the <abbr>workspace</abbr>. The comparison can be narrowed down to
-specific target files/directories tracked by DVC.
+Show added, modified, or deleted DVC-tracked files and directories between
+commits in the <abbr>DVC repository</abbr>, or between a commit and the
+workspace.
 
 ## Synopsis
 
 ```usage
-usage: dvc diff [-h] [-q | -v] [-t TARGET] a_ref [b_ref]
+usage: dvc diff [-h] [-q | -v]
+                [--show-json] [--show-hash]
+                [a_rev] [b_rev]
 
 positional arguments:
-  a_rev                 Old Git commit to compare (defaults to HEAD)
-  b_rev                 New Git commit to compare (defaults to the
-                        current workspace)
+  a_rev      Old Git commit to compare (defaults to HEAD)
+  b_rev      New Git commit to compare (defaults to the current workspace)
 ```
 
 ## Description
 
-Given two commit hashes, branch or tag names, etc.
-([references](https://git-scm.com/docs/revisions)) `a_ref` and `b_ref`, this
-command shows a comparative summary of basic statistics related to files tracked
-by DVC: how many files were deleted/changed, and the file size differences.
+Prints files and directories added, modified, deleted in a Git commit `b_rev` as
```

Comment:
Prints files sounds like maybe it `cat`s the file contents to me. That's why I suggested "Prints a list of files..."

Reply:
yep, good!